src/views/v2/components/icons/list.php

```diff
@@ -18,6 +18,12 @@
 
 if ( ! empty( $classes ) ) {
 	$svg_classes = array_merge( $svg_classes, $classes );
+	$id_prefix   = array_shift( $classes );
+} else {
+	$id_prefix = 'tribe-common-c-svgicon__list-clip-path-' . rand();
 }
+
+$clip0 = $id_prefix . '-0';
+$clip1 = $id_prefix . '-1';
```

Comment:
```suggestion
if ( ! empty( $classes ) ) {
	$svg_classes = array_merge( $svg_classes, $classes );
	$id_prefix   = array_shift( $classes );
} else {

	$id_prefix = 'tribe-common-c-svgicon__list-clip-path-';
	$id_prefix .= tribe_get_next_cached_increment( $id_prefix );
}

$clip0 = $id_prefix . '-0';
$clip1 = $id_prefix . '-1';
```


Place this function in tribe common in `src/functions/utils.php`:

```
/**
 * Get the next increment of a cached incrementable value.
 *
 * @since TBD
 *
 * @param string $key Cache key for the incrementor.
 * @param string $expiration_trigger The trigger that causes the cache key to expire.
 * @param int $default The default value of the incrementor.
 *
 * @return int
 **/
function tribe_get_next_cached_increment( $key, $expiration_trigger = '', $default = 0 ) {
	$cache = tribe( 'cache' );

	$value = (int) $cache->get( $key, $expiration_trigger, $default );
	$value++;
	$cache->set( $key, $value, \Tribe__Cache::NON_PERSISTENT, $expiration_trigger );

	return $value;
}
```